magento/pwa-studio
# [feature] Always show a wishlist name and do not show visibility toggle if only one list

## Description

* Added placeholder text for display if there is no wishlist name configured.
* Show or hide visibility toggle based on number of wishlists belonging to a user.

## Related Issue





Closes [PWA-1742](https://jira.corp.magento.com/browse/PWA-1742).

## Acceptance



### Verification Stakeholders

### Specification



### Verification Steps


1. Verify a wishlist without a name at least has the default "Wish List" name text.
2. In the user's wishlist page, verify that the visibility toggle does not appear if there is only one list.

## Screenshots / Screen Captures (if appropriate)

![](https://i.gyazo.com/c0595bc5668b56bf1ac007c71ab5c2f7.png)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
